todo_app/selenuim_test

```diff
@@ -0,0 +1,12 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+user_name = "jeremybrex@gmail.com"
+password = "Namaste123"
```

Comment:
Is this an actual password? If it is we should definitely remove this as well, and you should rotate this password for any systems you use it for

Reply:
of course not :-)

Reply:
I have removed the file. This was part of m3 testing